cmd/weaver-kube/deploy.go

```diff
@@ -49,30 +49,32 @@ Flags:
   -h, --help	Print this help message.
 
 Container Image Names:
-  "weaver kube deploy" builds and uploads a container image. You need to
-  specify the name of the container using the "image" field inside the "kube"
-  section of the config file. For example, consider the following config file:
+  "weaver kube deploy" builds a container image locally, and optionally uploads
+  it to a container repository. The repository can be specified using the
+  "repo" field inside the "kube" section of the config file. For example,
+  consider the following config file:
 
       [serviceweaver]
       binary = "./foo"
 
       [kube]
-      image = "docker.io/my_docker_hub_username/foo"
-
-  Using this config file, "weaver kube deploy" will build a container called
-  "docker.io/my_docker_hub_username/foo" and upload it to Docker Hub. The
-  format of the "image" field depends on the registry being used. Some
-  examples:
-
-      - Docker Hub: USERNAME/NAME or docker.io/USERNAME/NAME
-      - Google Artifact Registry: LOCATION-docker.pkg.dev/PROJECT-ID/REPOSITORY/NAME
-      - GitHub Container Registry: ghcr.io/NAMESPACE/NAME
-
-  "weaver kube deploy" will automatically append a tag to the image name, so
-  the "image" field should not contain a tag.
-
-  Note that for "weaver kube deploy" to work correctly, you must be
-  authenticated with the provided registry (e.g., by running "docker login".)`,
+      tag  = "foo/foo:0.0.1"
+      repo = "docker.io/my_docker_hub_username/my_repo"
+
+  Using this config file, "weaver kube deploy" will build a container
+  with a local tag "foo/foo:0.0.1", and upload it to the
```

Comment:
nit: 
can you add a reference to the docker tag `with a local tag [1] ...
....
[1] https://docs.docker.com/engine/reference/commandline/tag/
`